engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -150,6 +151,21 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
             pyEnv.setFailOnInitialize(false);
         }
 
+        // Handle TRT-LLM
+        if ("TRT-LLM".equals(Utils.getenv("LMI_BACKEND")) || Boolean.parseBoolean(getProperty("option.trt_llm"))) {
+            try {
+                Path trtLlmRepoDir = TrtLLMUtils.initTrtLlmModel(this);
+                if (trtLlmRepoDir != null) {
+                    String modelId = trtLlmRepoDir.toAbsolutePath().toString();
+                    setProperty("model_id", modelId);
+                    pyEnv.addParameter("model_id", modelId);
+                    entryPoint = "djl_python.tensorrt_llm";
```

Comment:
does this block setting custom entrypoint set by the user?

Reply:
Yes. If that isn't desired, you can move it back or set only if entryPoint is null